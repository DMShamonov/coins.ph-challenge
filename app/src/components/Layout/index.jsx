// @flow

import * as React from 'react';

export default function Layout(): React.Node {
  return (
    <div className="app">
      <header className="app__header">
        Header
      </header>
      <main className="app__main">
        Main
      </main>
    </div>
  );
}
