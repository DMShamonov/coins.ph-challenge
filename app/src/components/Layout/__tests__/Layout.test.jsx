import React from 'react';
import { shallow } from 'enzyme';

import Layout from '../index';

describe('`Layout` component should render correctly', () => {
  test('with default props', () => {
    expect(shallow(<Layout />)).toMatchSnapshot();
  });
});
