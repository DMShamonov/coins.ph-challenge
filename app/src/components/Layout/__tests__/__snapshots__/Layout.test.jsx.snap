// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`\`Layout\` component should render correctly with default props 1`] = `
<div
  className="app"
>
  <header
    className="app__header"
  >
    Header
  </header>
  <main
    className="app__main"
  >
    Main
  </main>
</div>
`;
